x-pack/plugins/security/server/config_deprecations.ts

```diff
@@ -122,6 +122,7 @@ export const securityConfigDeprecationProvider: ConfigDeprecationProvider = ({
         title: i18n.translate('xpack.security.deprecations.maxRedirectURLSizeTitle', {
           defaultMessage:
             '"xpack.security.authc.providers.saml.<provider-name>.maxRedirectURLSize" has no effect',
+          ignoreTag: true,
```

Comment:
Im not sure I verified this correctly, but should these still get translated?
<img width="1512" alt="Screenshot 2024-06-07 at 12 34 12 PM" src="https://github.com/elastic/kibana/assets/21210601/dfe7eb62-734e-4646-8c31-2f8fd79ee2dc">

Reply:
I only added the `ignoreTag: true` property which tells i18n that to ignore the XML tag inside the message instead of treating it as HTML and look for its closing tag.

So technically what you need to be testing is that the string is showing in the screen (which it does).

It seems we do not have a translation for this string in other languages yet which is OK.
Our translations to other languages are extracted and merged into Kibana periodically.

In between translation cycles any missing translations will show the `defaultMessage` in english like this case.

Reply:
++ thank you!